portfolio/src/main/webapp/js/maps.js

```diff
@@ -59,3 +66,22 @@ function createUFOMap() {
       new google.maps.Marker({position: {lat: sighting.lat, lng: sighting.lng}, map: ufoMap});
     }));
 }
+
+/** Display a simple hard-coded geomap using Charts API. */
+function drawRegionsMap() {
+  var data = google.visualization.arrayToDataTable([
+    ['Country', 'Popularity'],
+    ['Germany', 200],
+    ['United States', 300],
+    ['Brazil', 400],
+    ['Canada', 500],
+    ['France', 600],
+    ['RU', 700]
```

Comment:
why is Russia abbreviated?

Reply:
This was given as an example hard-coded data. It's modified.